samples/loadimpactair_script_sample.js

```diff
@@ -0,0 +1,403 @@
+ * stages: from 0 to 10 vu in 20 seconds
+ * 		   from 10 to 15 vu in 20 seconds
+ * 		   from 15 to 20 vu in 20 seconds
+ * 		   stay at 20 vu for 40 seconds
+ * 		   from 20 to 80 vu in 20 seconds
+ * 		   stay at 80 vu for 180 seconds
+ * 	  	   from 80 to 0 vu in 10 seconds
+ */
+export let options =  {
+	vusMax: 100,
+	duration: "10m",
+	stages: 
+		[{target: 10, duration: "20s"},
+		 {target: 15, duration: "20s"},
+		 {target: 20, duration: "20s"},
+		 {target: 20, duration: "40s"},
+		 {target: 80, duration: "20s"},
+		 {target: 80, duration: "180s"},
+		 {target: 0, duration: "10s"}],	
+	maxRedirects: 15,
+	insecureSkipTLSVerify: true,
+}
+
+
+// Create new trend metric named "group"
+var tr = new Trend("group");
+
+// Use a top-level function wrapper to allow us to return from it. If you don't need to
+// return from your script at any point, you can skip the wrapper - VU scopes are isolated.
+export default function() {
+
+	// Something to hold the responses in
+	var res = null;
+	// Timer start time
+	var tStart = 0;
+	// Timer end time
+	var tEnd = 0;
+
+	// First step - go to the landing page
+	// Put in group and name group something resaonable
+	// Batches are from how the real page is actually loaded
+	//   first the page then the next level content
+	//   and the last batch is the content from the bottom level of the content hierarchy
+	group ("01_landingpage", function () {
+
+		// Collect start time of group execition
+		tStart = (new Date()).getTime();
+
+		http.batch([
+			{"method" : "GET", "url" : "http://loadimpactair.guldskeden.se/"}
+		]);
+		http.batch([
+			{"method" : "GET", "url" : "http://loadimpactair.guldskeden.se/dtagent630_q_1268.js"},
+			{"method" : "GET", "url" : "http://loadimpactair.guldskeden.se/css/Style.css"},
+			{"method" : "GET", "url" : "http://loadimpactair.guldskeden.se/bundles/modernizr?v=wBEWDufH_8Md-Pbioxomt90vm6tJN2Pyy9u9zHtWsPo1"},
+			{"method" : "GET", "url" : "http://loadimpactair.guldskeden.se/Content/css?v=PUDFxlRUUS8e8pp6Y9WeVnF_4RmJM7BwtYyTz0D-zu81"},
+			{"method" : "GET", "url" : "http://loadimpactair.guldskeden.se/Media/Images/milehightheader.jpg"},
+		]);
+		http.batch([
+			{"method" : "GET", "url" : "http://loadimpactair.guldskeden.se/Media/Images/pattern.png"}
+		]);
+
+		// Collect end time of execution
+		tEnd = (new Date()).getTime();		
+	});
+
+	// Add trend metric data point,
+	// i.e. the time it took the group to execute
+	// and name the data point the same name as the group name
+    tr.add(tEnd - tStart, {tag: "01_landingpage"});
+
+	// Sleep between 5-10 seconds (the amount of time an average user would read before the next step/action)
+	sleep(rnd(5, 10));
+
+	// Second step - go to booking page
+	// Put in group and name group something resaonable
+	// Keep response of reqest in res variable
+	group ("02_toBooking", function () {
+
+		// Collect start time of group execition
+		tStart = (new Date()).getTime();
+
+		res = http.batch([

```

Comment:
A single-URL batch is redundant. Also, add some checks to it - you wanna know if the server starts erroring.